Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -174,21 +173,48 @@ void TilesetContentManager::loadTileContent(
           _externals.pAssetAccessor,
           _externals.pLogger,
           _requestHeaders)
-      .thenInWorkerThread(
+      .thenImmediately(
```

Comment:
task returned by the loader will be run immediately, so that we can determine if we should spawn a new worker thread for render content. If the result is not a render content and not success, it can end up in the main thread much faster if the previous task is ran on main thread for some reasons